doc/DHCPv6_relay/DHCPv6-relay-agent-High-Level-Design.md

```diff
@@ -223,6 +286,8 @@ Configurable option to use loopback address for dual ToR
 # Performance
 
 SONiC DHCP relay agent is currently not relaying many DHCP requests. Frequency arrival rate of DHCP packets is not high so it is not going to affect performance.
+Typical number of vlans used with dhcp6relay is around 2 vlans. Performance degration and high CPU utilization is seen on devices with more than 25 vlans configured, especially devices with weaker CPU's.
```

Comment:
I'd suggest adding a section called scalability and mention the typical use case for dhcpv6 relay and ideally the maximum number of vlans this feature was ever tested on. 

Then followed by a point specifying functionality and performance degradation when there are > 20 vlans configured on weaker CPU's

Reply:
Technically, it's currently broken for more than 1 vlan. Issue: https://github.com/sonic-net/sonic-buildimage/issues/12805. But i'm assuming this'll be fixed at the earliest and it's okay to merge the PR with these stats.